src-java/floodlightrouter-topology/floodlightrouter-storm-topology/src/main/java/org/openkilda/wfm/topology/floodlightrouter/RegionAwareKafkaTopicSelector.java

```diff
@@ -0,0 +1,64 @@
+/* Copyright 2020 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.wfm.topology.floodlightrouter;
+
+import joptsimple.internal.Strings;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.storm.kafka.bolt.selector.KafkaTopicSelector;
+import org.apache.storm.tuple.Tuple;
+
+@Slf4j
+public class RegionAwareKafkaTopicSelector implements KafkaTopicSelector {
+    public static final String FIELD_ID_TOPIC = "topic";
```

Comment:
would be nice to see this tuple field names in a single place someday(not in this pr)

Reply:
? Why for? 
Any bolt that wants to use this topic-selector must use this filed names because the topic selector expects them in the tuple. So bolts can refer to these constants.

This class is responsible for these fields, it defines the logic of their usage and it requires their presence into the tuple. So why we need some 3rd place to define these names/constants?